src/css/objects/_media.scss

```diff
@@ -245,172 +241,216 @@
 // We're bending some BEM rules here by having 1-deep nested selectors, but in
 // exchange for that we get a single common markup pattern for media objects.
 
-.media--last {
-  margin-bottom: 0;
-}
-
-.media--new-and-unread {
-
-  .media__headline {
-    position: relative;
-
-    &::before {
-      background-color: $color-primary;
-      border-radius: 50%;
-      content: '';
-      display: block;
-      height: 0.5rem;
-      left: ($base-line-height / -2);
-      position: absolute;
-      top: 1rem;
-      width: 0.5rem;
-    }
+.o-media--sm {
+  align-items: center;
+  display: flex;
+  margin: 0 0 $base-line-height;
 
+  .o-media__headline, .o-media__title {
+    margin: 0;
   }
 
-  .media__indicator-visited {
-    display: none;
+  > .o-media__figure {
+    margin-right: 1.125rem;
+    max-width: 5rem;
+    float: left;
+    margin-right: 3.55505%;
+    width: 4.5rem;
   }
 
+  .o-media__bylineText {
+    border-bottom: 4px solid $color-gray-light;
+    padding-bottom: 0.5rem;
+  }
 }
 
-.media--hp {
-  box-shadow: inset 17rem 0 0 $color-gray-lightest;
-  padding: $base-line-height 0 $base-line-height 19rem;
+.o-media--med {
+  // box-shadow: inset 17rem 0 0 $color-gray-lightest;
+  // padding: $base-line-height 0 $base-line-height 19rem;
   position: relative;
 
+  //NOTE: Re-add the following padding as part of the CSS specific to the homepage rather than here.
+  // padding-top: 4.25rem
+
+  .o-media__figure {
+    margin-bottom: 1.375rem;    
+  }
+
   @include media-max($media-tablet) {
     border-top: 1px solid $color-gray-light;
     box-shadow: none;
     padding-left: 0;
-  }
 
-  &::after {
-    border-top: 1px solid $color-gray-light;
-    content: '';
-    display: block;
-    left: 17rem;
-    position: absolute;
-    right: 0;
-    top: 0;
+    .o-media__content {
+      // If flex is available, and we are on mobile,
+      // use this to position the figure before the 
+      // rest of the media content.
+      display: flex;
+      flex-direction: column;
+      .o-media__figure {
+        order: -1
+      }
+    }
 
-    @include media-max($media-tablet) {
-      display: none;
+    .o-media__img {
+      margin: 0 auto;
     }
-  }
 
-  > .media__figure {
-    margin: 0 0 $base-line-height;
   }
 
-  > .media__headline {
-    margin: 0 0 $base-line-height;
-  }
+  @include media-min($media-tablet){
+    $width: 18.625rem;
 
-  > .media__label {
-    font-size: ms(-2);
-  }
+    [class*='o-media__figure--'] {
+      .o-media__img {
+        box-sizing: border-box;
+        left: 50%;
+        position: absolute;
+        top: 50%;
+        transform: translate(-50%, -50%);
+      }
+    }
 
-  .media__overline {
-    color: $color-secondary;
-  }
```

Comment:
Currently this float is only applied to tablet breakpoints and larger. It's worth testing out whether you can get away with continuing to float the image until you get down to our mobile breakpoint.